### PR TITLE
CD-675 AH fix integer alias

### PR DESCRIPTION
**Pull request by ahaessly**

### CODE REVIEW

File: src/main/java/bio/terra/cda/app/models/DataSetInfo.java [hunk 1]
```diff
@@ -89,7 +89,7 @@ public ColumnDefinition getColumnDefinitionByFieldName(String fieldName, String
     if (fieldName.contains(".")) {
       // it's a mapping field
       String[] parsed = fieldName.split("\\.", 2);
-      TableInfo mappingTable = mappingTableInfoMap.get(parsed[0]);
+      TableInfo mappingTable = getTableInfo(parsed[0]);
       return Arrays.stream(mappingTable.getColumnDefinitions()).filter(col -> col.getName().equals(parsed[1])).findFirst().orElse(null);
```
Comment (KutnerBroadie) [inline src/main/java/bio/terra/cda/app/models/DataSetInfo.java:93]: This isn't necessary at all, but I find this code a bit hard to follow. (For me) it would be easier to read if the filter conditional were parsed out into its own variable (if possible) so it read
".filter(filterCondition).findFirst().orElse(null);"

Are there conventions in Java for how to do this otherwise? Like whitespace or formatting?

File: src/test/java/bio/terra/cda/app/generators/EntityCountSqlGeneratorTest.java
Comment (KutnerBroadie) [inline src/test/java/bio/terra/cda/app/generators/EntityCountSqlGeneratorTest.java:1]: Just curious - how were these test queries built/generated? And what changed in them for this PR? I'm having trouble parsing through the giant text blocks. :)

**Review by KutnerBroadie — CHANGES_REQUESTED** [review]

File: src/main/java/bio/terra/cda/app/models/DataSetInfo.java (hunk 1 above)
Comment (fthomas641) [reply src/main/java/bio/terra/cda/app/models/DataSetInfo.java:93]: One possibility for clarity would be for the "parsed" values to be more explicit. 
E.g. 
```
String tablename = parsed[0];
String colname = parsed[1];
```

I have seen some Java streams ops that use whitespace for clarity. E.g.:
```
List<Customer> charlesWithMoreThan100Points = customers
  .stream()
  .filter(c -> c.getPoints() > 100 && c.getName().startsWith("Charles"))
  .collect(Collectors.toList());
```

So in this case maybe more like:

```
Arrays.stream(mappingTable.getColumnDefinitions())
    .filter(col->col.getName().equals(parsed[1]))
    .findFirst().orElse(null);
```

Comment (KutnerBroadie) [reply src/main/java/bio/terra/cda/app/models/DataSetInfo.java:93]: I like both these suggestions. I think they make it look a lot easier to understand!

File: src/test/java/bio/terra/cda/app/generators/EntityCountSqlGeneratorTest.java
Comment (ahaessly) [reply src/test/java/bio/terra/cda/app/generators/EntityCountSqlGeneratorTest.java:1]: yeah, it's disappointing that the diff view doesn't highlight the differences. basically the update code has alias fields instead of id fields for some of the columns

**Review by fthomas641 — APPROVED** [review]
Looks good.

**Review by KutnerBroadie — APPROVED** [review]

**Review by tanner-coon-bh — APPROVED** [review]
Everything looks good to me.

**Comment by KutnerBroadie** [comment]
Rerun tests
